Assets/Scripts/Models/Area/RoomManager.cs

```diff
@@ -0,0 +1,429 @@
+﻿#region License
+// ====================================================
+// Project Porcupine Copyright(C) 2016 Team Porcupine
+// This program comes with ABSOLUTELY NO WARRANTY; This is free software, 
+// and you are welcome to redistribute it under certain conditions; See 
+// file LICENSE, which is part of this source code package, for details.
+// ====================================================
+#endregion
+using System;
+using System.Collections;
+using System.Collections.Generic;
+
+namespace ProjectPorcupine
```

Comment:
The naming conventions encourage namespaces to be plural unless it doesn't make sense. I vote for `ProjectPorcupine.Rooms`